experimental/apps-mcp/lib/trajectory/tracker.go

```diff
@@ -0,0 +1,133 @@
+package trajectory
```

Comment:
Non-blocking: types here should have comments describing what they're doing, they're non-obvious